### PR TITLE
feat: Login

### PR DESCRIPTION
**Pull request by tringuyen-dev**

### CODE REVIEW

File: src/Controller/User/ProfileController.php [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<?php
```
Comment (nhiv0) [inline src/Controller/User/ProfileController.php]: i think Profile is not enough to be a Controller.

File: src/Repository/.gitignore [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+/public/bundles/
+/var/
+/vendor/
+.phpunit.cache
+/.idea/
+*.phar
+.env
+.phpunit.result.cache
+psalm.xml
+###< symfony/framework-bundle ###
```
Comment (nhiv0) [inline src/Repository/.gitignore]: EOL

File: src/DataFixtures/AppFixtures.php [hunk 3]
```diff
@@ -11,7 +11,6 @@ public function load(ObjectManager $manager): void
     {
         // $product = new Product();
         // $manager->persist($product);
```
Comment (nhiv0) [inline src/DataFixtures/AppFixtures.php]: should remove unused code

**Comment by QuocThangPhu** [comment]
=I agree with Nhi about EOL and comment code

**Pull request merged**